tests/console/docker.pm

```diff
@@ -116,6 +116,18 @@ sub run {
         assert_script_run('man -P cat docker config | grep "docker-config - Manage Docker configs"');
     }
 
+    # Try to stop container using ctrl+c
+    type_string("docker run --rm opensuse/tumbleweed sleep 30\n");
+    type_string("# Let's press ctrl+c right now ... ");
+    send_key 'ctrl-c';
+    type_string("# ... and we seem to be still in container\n");
+    # If echo works then ctrl-c stopped sleep
+    type_string "echo 'ctrlc_timeout' > /dev/$serialdev\n";
+    if (wait_serial 'ctrlc_timeout', 10) {
```

Comment:
wait_serial has extra parameter `$expect_not_found`, when set to 1 it won't fail when match is not found and won't create red box (visible in your local runs).
Check documentation - http://open.qa/api/testapi/#_wait_serial

Reply:
It should be fixed now, but we're having technical issues with online repositories so I can't run it in openQA now...